docs/architecture-decision-records/0018-ingest-records-as-assets-not-folders.md

```diff
@@ -0,0 +1,31 @@
+# 18. Ingest records as assets, not folders
+
+**Date:** 2024-06-17
+
+## Context
+
+When ingesting to the Preservation System we have a choice to represent the record as an asset or a folder. Many archives choose to represent records as a folders within their Catalogue and Preservation System, removing the need to describe individual objects within a collection.

```

Comment:
1. Missing comma after "When ingesting to the Preservation System"?
2. Should "asset" and "folder" be capitalised?
3. "as a folders"; is it "as folders" or "as a folder"?